view/frontend/layout/checkout_index_index.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

Comment:
Not a good idea to depend on default, 
It will be good to use,  layout="checkout"

Reply:
Updated with checkout layout.